pkg/network/server.go

```diff
@@ -233,16 +252,65 @@ func (s *Server) run() {
 	}
 }
 
+func (s *Server) tryStartConsensus() {
+	if s.Wallet == nil || s.connected.Load() {
+		return
+	}
+
+	myAddr := net.JoinHostPort(s.Address, strconv.Itoa(int(s.Port)))
+	connected := make([]bool, len(s.Seeds))
+	seedCount := 0
+
+	log.Infof("seeds: %v", s.Seeds)
+	for p := range s.Peers() {
+		log.Infof("array: %v", connected)
+		rAddr := p.RemoteAddr().String()
+		pAddr := p.PeerAddr().String()
+		log.Infof("my: %s, remote: %s %s", myAddr, rAddr, pAddr)
+		for i, seed := range s.Seeds {
```

Comment:
I don't think it's correct to check our connectivity against seeds. Seeds may be unconnected and that's perfectly fine for a node. In mainnet you don't even get a lot of chance to connect to seed nodes, they're usually at the max of their allowed peer count.

Reply:
This was to fix some problems regarding starting consensus nodes.
After some investigation I found out that changes in `docker-compose` can also help that.